### ISSUE TITLE
Documentation on how to use PyWebScrapBook with a WGSI server like mod_wsgi

### ISSUE THREAD

**Issue by jdunn0** [danny0838/PyWebScrapBook#63]
I want to use PyWebScrapBook with mod_wsgi but I'm not sure how to do so.

According to the [Quick Configuration Guide for mod_wsgi](https://modwsgi.readthedocs.io/en/master/user-guides/quick-configuration-guide.html), I would use the directive `WSGIScriptAlias /myapp /usr/local/www/wsgi-scripts/myapp.wsgi` to have /myapp load the script myapp.wsgi in Apache.

The issue is that I don't know what the script file name would be for PyWebScrapBook. When I look at the [PyWebScrapBook/webscrapbook](https://github.com/danny0838/PyWebScrapBook/tree/master/webscrapbook) directory here on Github, it shows the files `__init__.py, __main__.py, app.py, cli.py, locales.py and server.py`.

Because the app.py file has a comment that says "The WGSI application.", I would guess that app.py is the file I need to tell mod_wgsi to load but that could just be an included file and one of the other script files are what I need to specify in the directive.

It be helpful to me and possibly others to put in the documentation wiki, a part about using a WSGI server that lists what script file name to use along with any other information that might be helpful

**Comment by danny0838** [comment]
You need to first generate config files as the `Host a scrapbook` section mentioned, and provide the **generated** `app.py` to the upper server.

**Comment by jdunn0** [comment]
Thanks, I had misunderstood the instructions before and didn't realize I needed to run the config function to install scripts in the .wsb directory to call the built-in server (serve.py) or WSGI server (app.py).

I had assumed that I would call the scripts in the installation directory (`/usr/lib/python3.10/site-packages/webscrapbook` on my server).

Thus why I opened bug #61 a bit ago, I was assuming that I would be running a script from the the `site-packages/webscrapbook` directory which would not be a good place to put my ScrapBook directory.

This is an unusual and unexpected way for the software to work, as configuration directories don't usually contain scripts to run the software but it works which is good as I've been wanting to try WebScrapBook as I manged to use Waterfox Classic;/Palemoon with ScrapBook X for a while now but in the past year or so, I've been finding more web sites that just don't work right in those browsers which sucks.

I think it would be worth appending the "Host a scrapbook" section to mention that app.py is available for use with WSGI servers as it currently only mentions running serve.py or running `wsb serve`.
